packages/web/src/components/artist/ArtistChipFollowers.tsx

```diff
@@ -34,6 +34,7 @@ export const ArtistChipFollowers = ({
       </div>
       {doesFollowCurrentUser ? (
         <FollowsYouBadge
+          transparentBackground
```

Comment:
this kind of a prop feels like css in js 😅 

maybe we should move the `ArtistChip` specific styles into a "variant" that has a transparent background?

Reply:
variant good idea